pandas/tests/scalar/timedelta/test_arithmetic.py

```diff
@@ -616,3 +680,14 @@ def test_rdivmod_invalid(self):
 
         with pytest.raises(TypeError):
             divmod(np.array([22, 24]), td)
+
+    @pytest.mark.parametrize('op', [operator.mul, ops.rmul, operator.truediv, ops.rdiv])
+    @pytest.mark.parametrize('arr', [
+                                      [Timestamp('20130101 9:01'),
+                                       Timestamp('20121230 9:02')],
+                                      [pd.Timestamp.now(),
+                                       pd.Timedelta('1D')]
+                                    ])
+    def test_td_op_timedelta_timedeltalike_array(self, op, arr):
+        with pytest.raises(TypeError):
+            arr * pd.Timedelta('1D')
```

Comment:
You're going to need one more newline at the end of this file.

Reply:
good catch, done